packages/o11y-codec-rt/alp/src/lib.rs

```
@@ -111,6 +127,831 @@ pub fn alp_try(val: f64, e: usize) -> Option<i64> {
+    if is_delta {
+        decode_delta_for_exceptions(
+            input, pos, header_u64, actual_bw, exc_count, n, &exc_positions, val_out,
+        );
+    } else {
+        decode_for_exceptions(
+            input, pos, header_u64, actual_bw, exc_count, n, &exc_positions, val_out,
+        );
+    }
+}
+
+fn decode_delta_for_exceptions(
```

Clippy (1.95.0)

[warning] this function has too many arguments (8/7)

(warning)